### ISSUE TITLE
Reproduction of the results of the paper

### ISSUE THREAD

**Issue by M-o-magic** [YuyangSunshine/ABR_IOD#12]
Hello, I ran first run_firststep.sh then run scripts/run_SI.sh under the setting of 10-10, and the result was 2.0 lower than the result in your paper, do you know the reason?

run_firststep.sh
```
#!/bin/bash
port=$(python tools/get_free_port.py)
GPU=8
CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7
export CUDA_HOME=/usr/local/cuda-11.3
export PATH=$CUDA_HOME/bin:$PATH
export LD_LIBRARY_PATH=$CUDA_HOME/lib64:$LD_LIBRARY_PATH
#  Choose the corresponding first task #
# 19 classes in first task #
#task=19-1

# 15 classes in first task #
# task=15-5

# 10 classes in first task #
task=10-10

# 5 classes in first task #
# task=5-15


#### 1. Train the First Task ####

python -m torch.distributed.launch --master_port=${port} --nproc_per_node=${GPU} tools/train_first_step.py -c configs/voc/${task}/e2e_faster_rcnn_R_50_C4_4x.yaml -cvd ${CUDA_VISIBLE_DEVICES}

#### 2. Prototype Box Selection (PBR) ####
step=0
name=ABR # The corrensponding name for the incremental method
python tools/prototype_box_selection.py \
    -cvd 0 \
    -n ${name} -t ${task} -s ${step} \
    -mb 2000 \
    -mt mean \
    -iss

```

run_SI.sh 
```
port=$(python tools/get_free_port.py)
GPU=8

alias exp="python -m torch.distributed.launch --master_port=${port} --nproc_per_node=${GPU} tools/train_incremental.py"
shopt -s expand_aliases


# INCREMENTAL STEPS
step=1

#task=19-1
#name=ABR_LR001_BS4_ALPHA1_BETA1_GAMMA5
#exp -t ${task} -n ${name} -s ${step} --feat ard -gamma 5.0 --uce --dist_type id -alpha 1.0 -beta 1.0 -mb 2000 -mt mean -cvd 0,1
#name=Finetune
#exp -t ${task} -n ${name} -s ${step} -cvd 0,1


# task=15-5
task=10-10
name=ABR_LR001_BS4_ALPHA05_BETA1_GAMMA1
exp -t ${task} -n ${name} -s ${step} --feat ard -gamma 1.0 --dist_type id -alpha 0.5 -beta 1.0 -mb 2000 -mt mean -cvd 0,1,2,3,4,5,6,7
```

**Comment by YuyangSunshine** [comment]
Hi, thanks for your attention!

If the underlying GPU hardware settings are different, it may affect the final results. 
Perhaps you can try to slightly adjust the basic hyperparameters (learning rate, epoch, etc.) to suit your device.
